python-manual/modules/ROOT/pages/performance.adoc

```diff
@@ -0,0 +1,201 @@
+= Performance recommendations
+
+// users ask info on how to handle high throughput
+
+- *Specify the target database on all queries* with the `database` parameter when creating new xref:transactions.adoc[sessions]. If no database is provided, the driver has to send an extra request to the server to figure out what the default database is. The overhead is minimal for a single session, but becomes significant over hundreds of sessions.
++
+[source,python,role=noheader]
+----
+# Good practice
+driver.session(database="<DB NAME>")
+----
++
+[source,python,role=noheader]
+----
+# Bad practice
+driver.session()
+----
+
+// - Execute as many queries within the same session. Minimise the number of transactions?
```

Comment:
Yeah, they are related. Although I see it as slightly different, in that the note after is about batching creation, while we also want to encourage general parsimony in transactions. They boil down to the same principle, but it might not be so apparent to new users, so I wanted to be extra clear that it is meaningful also in general, not just when inserting lots of data.

Reply:
Fair points. It's also just a comment. Let me highlight again, though, that using fewer sessions doesn't give you much (other than more automatic causal chaining which might even end up slightly slowing down the application if it's not needed). You can resolve this comment, when you've read this.